src/jit/ee_il_dll.cpp

```diff
@@ -163,6 +163,20 @@ CorJitResult CILJit::compileMethod (
         return g_realJitCompiler->compileMethod(compHnd, methodInfo, flags, entryAddress, nativeSizeOfCode);
     }
 
+    if (!JitConfig.isInitialized())
```

Comment:
There's an interesting assumption here that any compHnd can be used to initialize JitConfig, exactly once. Subsequent calls to compileMethod() don't need to reinitialize, and we assume that every ICorJitInfo would have returned the same data. This is probably true for this case, but maybe a note needs to be made (in the interface header?) about this assumption.